src/Microsoft.Identity.Web/MicrosoftIdentityCircuitHandler.cs

```diff
@@ -17,257 +17,257 @@
+				+ $"&{Constants.DomainHintParameter}={User.GetDomainHint()}&{Constants.Claims}={claims}"
+				+ $"&{OidcConstants.PolicyKey}={userflow}";
 
-            if (IsBlazorServer)
-            {
-                NavigationManager.NavigateTo(url, true);
-            }
-            else
-            {
+			if (IsBlazorServer)
+			{
+				NavigationManager.NavigateTo(url, true);
+			}
+			else
+			{
 #pragma warning disable CS8602 // Dereference of a possibly null reference. HttpContext will not be null in this case.
-                _httpContextAccessor.HttpContext.Response.Redirect(url);
+				_httpContextAccessor.HttpContext.Response.Redirect(url);
 #pragma warning restore CS8602 // Dereference of a possibly null reference. HttpContext will not be null in this case.
-            }
-        }
+			}
+		}
 
-        internal NavigationManager NavigationManager { get; set; } = null!;
-    }
+		internal NavigationManager NavigationManager { get; set; } = null!;
+	}
 
 #pragma warning disable SA1402 // File may only contain a single type
-    internal class MicrosoftIdentityServiceHandler : CircuitHandler
+	internal class MicrosoftIdentityServiceHandler : CircuitHandler
 #pragma warning restore SA1402 // File may only contain a single type
-    {
-        public MicrosoftIdentityServiceHandler(MicrosoftIdentityConsentAndConditionalAccessHandler service, AuthenticationStateProvider provider, NavigationManager manager)
-        {
-            Service = service;
-            Provider = provider;
-            Manager = manager;
-        }
+	{
+		public MicrosoftIdentityServiceHandler(MicrosoftIdentityConsentAndConditionalAccessHandler service, AuthenticationStateProvider provider, NavigationManager manager)
+		{
+			Service = service;
+			Provider = provider;
+			Manager = manager;
+		}
 
-        public MicrosoftIdentityConsentAndConditionalAccessHandler Service { get; }
+		public MicrosoftIdentityConsentAndConditionalAccessHandler Service { get; }
 
-        public AuthenticationStateProvider Provider { get; }
+		public AuthenticationStateProvider Provider { get; }
 
-        public NavigationManager Manager { get; }
+		public NavigationManager Manager { get; }
 
-        public override async Task OnCircuitOpenedAsync(Circuit circuit, CancellationToken cancellationToken)
-        {
-            var state = await Provider.GetAuthenticationStateAsync().ConfigureAwait(false);
-            Service.User = state.User;
-            Service.IsBlazorServer = true;
-            Service.BaseUri = Manager.BaseUri.TrimEnd('/');
-            Service.NavigationManager = Manager;
-            await base.OnCircuitOpenedAsync(circuit, cancellationToken).ConfigureAwait(false);
-        }
-    }
+		public override async Task OnCircuitOpenedAsync(Circuit circuit, CancellationToken cancellationToken)
+		{
+			var state = await Provider.GetAuthenticationStateAsync().ConfigureAwait(false);
+			Service.User = state.User;
+			Service.IsBlazorServer = true;
```

Comment:
IsBlazorServer might not be correct or am I reading this wrong?

Reply:
this is only for blazorwasm...it's a long story.